kubernetes/README.md

```diff
@@ -0,0 +1,86 @@
+# Deploying OpenWhisk on Kubernetes
+
+[![Build Status](https://travis-ci.org/openwhisk/openwhisk-devtools.svg?branch=master)](https://travis-ci.org/openwhisk/openwhisk-devtools)
+
+The collection of scripts and Kubernets pods can be used to
+run OpenWhisk in a Kubernets cluster.
+
+#### Kubernetes Requirments
+
+* Kubernetes needs to be version 1.5+
+* Kubernetes pods can receive public addresses
```

Comment:
This one is a bit tricky to try and explain as it deals with cluster setup. For example, if I am running kube locally on a Ubuntu machine, then I can access my Kube Pods because I am able to share the network. However, if say you are on a hosted environment Softlayer, AWS, GCE then what you can do is create a `LoadBalancer` type of service. This will then have Kubernetes communicate  with the provider to and actually create a Load Balancer and map that IP address to a Kubernetes service.

For other solutions where there is no LoadBalancer, then routers need to be manually configured to communicate with Kubernetes nodes. Even then there are multiple solutions on how to do this depending on what Cloud Network Interface is being used in Kubernetes. For example, Callico could be configured like [this](https://github.com/projectcalico/calicoctl/blob/v0.19.0/docs/ExternalConnectivity.md).

Right now, I'm cleaning up the README to be more clear and providing instructions for how I am running the environment to test on locally.